tsconfig.json

```diff
@@ -1,17 +1,13 @@
 {
-    "compilerOptions": {
-        "target": "es5",
-        "module": "es2015",
-        "declaration": true,
-        "outDir": "./dist",
-        "strict": true,
-        "lib": [
-            "es6",
-            "dom"
-        ],
-        "noImplicitAny": false,
-        "resolveJsonModule": true,
-        "esModuleInterop": true,
-        "moduleResolution": "node"
-    }
+  "include": ["./lib/**/*.ts"],
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "moduleResolution": "node",
+    "target": "ESNext",
```

Comment:
This is handled by rullup ts plugin @artonge 

```ts
typescript({
  tsconfig: './tsconfig.json',
  compilerOptions: { target: 'es5' },
})
```

![image](https://user-images.githubusercontent.com/1561347/165963446-da36dcbb-0d75-4c3d-a3a2-2c6d81034027.png)

This is the default bundle target.

Reply:
It will probably not go through babel again because we exclude node_modules by default. I think we should keep targetting ES6 here.

Reply:
Would it be possible to put the target into `tsconfig.json` directly? This would certainly reduce confusion.

Reply:
Now this package generates 2 output files.
- `dist/index.js` -> previous commonjs target, also es5 compliance.
- `dist/index.esm.js` -> new esm target, without es5 transpiling.

And it is addressed in `package.json`

```json
"main": "dist/index.js",
"module": "dist/index.esm.js",
"types": "dist/index.d.ts",
"exports": {
  "import": "./dist/index.esm.js",
  "require": "./dist/index.js"
},
```

Webpack, rollup, vite, parcel, skypack and any other bundle/builder can now read these options and choose the better target option.

Even an older bundler will keep everything working because we steel using commonjs as default.